README.md

```diff
@@ -35,12 +35,26 @@ The `install-dev.sh` script will build and push an image of the Tekton Dashboard
 
 1. Install [tektoncd-pipeline-operator](https://github.com/openshift/tektoncd-pipeline-operator#deploy-openshift-pipelines-operator-on-minikube-for-testing)
 2. [Checkout](https://github.com/tektoncd/dashboard/blob/master/DEVELOPMENT.md#checkout-your-fork) the repository
+
+If you want to install the dashboard into tekton-pipelines namespace:
 3. Install deployment config `oc process -f config/templates/deploy.yaml | oc apply -f -`
 4. Install build config `oc process -f config/templates/build.yaml | oc apply -f -`
 5. Wait until the pod `tekton-dashboard-1` is running in the tekton-pipelines namespace
 
+If you want to install the dashboard into any other namespace:
+3. Change line 47 in config/templates/deploy.yaml to `namespace: INSTALL_NAMESPACE`
+4. Run the command `chmod +x ./config/templates/minishiftInstallDashboard.sh`
```

Comment:
We can commit this chmodded

I don't think it should be in the templates directory, we already have install-dev.sh at the top level directory so it can go there

Let's use the same file name convention e.g. foo-bar.sh instead of camel case